firefox-ios/firefox-ios-tests/Tests/ClientTests/TelemetryContextualIdentifierTests.swift

```diff
@@ -49,11 +49,6 @@ class TelemetryContextualIdentifierTests: XCTestCase {
 
     // MARK: Helper methods
     func clearTest() {
-        // Due to changes allow certain custom pings to implement their own opt-out
-        // independent of Glean, custom pings may need to be registered manually in
-        // tests in order to put them in a state in which they can collect data.
-        Glean.shared.registerPings(GleanMetrics.Pings.shared)
-        Glean.shared.resetGlean(clearStores: true)
```

Comment:
I think `Glean.shared.resetGlean(clearStores: true)` sets Glean into "testing" mode if I remember properly. This test suite is calling `TelemetryWrapper.shared.setup(profile: MockProfile())` inside `testTelemetryWrapper_setsContextId`, which setups Glean. So the test might pass, but Glean is not in testing mode? I just want to double-check this is fine

Reply:
It should still be fine. (Sorry, I found my response in a tab but failed to click "comment" button :( )